package.json

```diff
@@ -128,6 +128,7 @@
 		"prettier": "npm:wp-prettier@2.0.5",
 		"progress-bar-webpack-plugin": "2.1.0",
 		"promptly": "3.2.0",
+		"puppeteer": "3.0.0",
```

Comment:
One caution, if we end up having to add puppeteer directly, we'll lose the benefits of "on demand" chromium download/update that we get from the current reliance on `wp-scripts` for puppeteer configuration/imports. This is significant in reducing the time `npm install` takes to complete (where chromium would end up getting downloaded always with this change).